test/ctia/test_helpers/benchmark.clj

```diff
@@ -1,30 +1,34 @@
 (ns ctia.test-helpers.benchmark
   (:require [clj-momo.lib.net :as net]
             [ctia
-             [init :refer [start-ctia!]]
-             [shutdown :as shutdown]]
+             [init :refer [start-ctia!]]]
             [ctia.test-helpers
              [core :as helpers]
-             [es :as esh]]))
+             [es :as esh]]
+            [puppetlabs.trapperkeeper.app :as app]))
 
 (defn setup-ctia! [fixture]
-  (let [http-port (net/available-port)]
-    (fixture (fn []
-               (helpers/fixture-properties:clean
-                #(helpers/with-properties ["ctia.store.es.default.refresh" "false"
-                                           "ctia.http.enabled" true
-                                           "ctia.http.port" http-port
-                                           "ctia.http.bulk.max-size" 100000
-                                           "ctia.http.show.port" http-port]
-                   (start-ctia!)))))
-    http-port))
+  (let [http-port (net/available-port)
+        app (fixture (fn []
+                       (helpers/fixture-properties:clean
+                         #(helpers/with-properties ["ctia.store.es.default.refresh" "false"
+                                                    "ctia.http.enabled" true
+                                                    "ctia.http.port" http-port
+                                                    "ctia.http.bulk.max-size" 100000
+                                                    "ctia.http.show.port" http-port]
+                            (start-ctia!)))))]
+    {:port http-port
+     :app app}))
 
 
 (defn setup-ctia-es-store! []
   (setup-ctia! esh/fixture-properties:es-store))
 
 (def delete-store-indexes esh/delete-store-indexes)
 
-(defn cleanup-ctia! []
+(defn cleanup-ctia! [app]
   (esh/delete-store-indexes false)
-  #(shutdown/shutdown-ctia!))
```

Comment:
The cleanup function doesn't return a function, this seems like an typo https://github.com/davidsantiago/perforate/blob/61bf670d806e6696785324d89e0fcac5ffad205d/src/perforate/core.clj#L22-L23